libpod/oci_conmon_exec_common.go

```diff
@@ -258,7 +258,7 @@ func (r *ConmonOCIRuntime) ExecStopContainer(ctr *Container, sessionID string, t
 
 	// SIGTERM did not work. On to SIGKILL.
 	logrus.Debugf("Killing exec session %s (PID %d) of container %s with SIGKILL", sessionID, pid, ctr.ID())
-	if err := pidHandle.Kill(unix.SIGTERM); err != nil {
+	if err := pidHandle.Kill(unix.SIGKILL); err != nil {
```

Comment:
I'm not sure if this is a mistake or if it was intentional. But the documentation string for this feature says: 
```
ExecStopContainer stops a given exec session in a running container. SIGTERM with be sent initially, then SIGKILL after the given timeout. If timeout is 0, SIGKILL will be sent immediately, and SIGTERM will be omitted.
```
So I would expect `uinx.SIGKILL` here instead of `unix.SIGTERM`.

Reply:
yes correct, that said it is not directly related to your change so I suggest you split out such change into its own commit.

Reply:
I moved the change to a separate commit.